engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -1569,4 +1569,20 @@ CREATE VIEW `cloud`.`user_view` AS
             left join
         `cloud`.`async_job` ON async_job.instance_id = user.id
             and async_job.instance_type = 'User'
-            and async_job.job_status = 0;
\ No newline at end of file
+            and async_job.job_status = 0;
+
+--- Create table for Virtual Machine Schedule Entity
+
+CREATE TABLE IF NOT EXISTS `cloud`.`vm_schedule` (
+  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `uuid` varchar(40) UNIQUE COMMENT 'UUID for the VM Schedule',
+  `name` varchar(255) COMMENT 'name of VM Schedule',
+  `action` varchar(40) NOT NULL COMMENT 'action Scheduled on VM',
+  `period` varchar(255) NOT NULL COMMENT 'period Scheduled on VM',
+  `timezone` varchar(40)  COMMENT 'timezone of VM',
+  `tag` varchar(40)  COMMENT 'Tag Value of VM Schedule',
+  `vm_id` bigint(20) unsigned NOT NULL COMMENT 'virtual machine id',
+  FOREIGN KEY (vm_id) REFERENCES `cloud`.`vm_instance`(`id`),
```

Comment:
tip - you may want an index on vm ID for quick search

Reply:
done